app.js

```diff
@@ -54,4 +66,4 @@ app.use(function(err, req, res, next) {
 
 module.exports = app
 
-app.listen(3000)
+app.listen(4000)
```

Comment:
At some point, I'd like to move this out into its own thing (similar to `config/config.json`) so that we can configure this via environment variable or something, but that's out of the scope of this PR.  Just a note-to-self.

Reply:
Yeah, for sure. I also think that we should consolidate all the config into one place, right now we have at least 2 different config files floating around.